src/backend/cli/utils.py

```diff
@@ -12,6 +23,43 @@ def show_welcome_message():
     )
 
 
+def process_existing_yaml_config(secrets, path, prompt):
+    _path = Path(path)
+
+    if _path.is_file():
+        try:
+            yaml_config = read_yaml(path)
+        except yaml.scanner.ScannerError:
+            if prompt():
+                yaml_config = {}
+                _path.unlink()
+            else:
+                sys.exit(1)
+
+        secrets.update(convert_yaml_to_secrets(yaml_config))
```

Comment:
Just want to make sure I understand the logic:

1. Earlier we load the `.env` values with `dotenv_values()`
2. Then we call this process function, which loads the YAML file (for either config/secrets)
3. YAML file is read and loaded
4. Iterate through each key of the .env <-> yaml mapping and if the value exists in the YAML file, we overwrite it for the secrets dictionary

Reply:
@tianjing-li yes this is right. The YAML values will overwrite any values read from the `.env` file first. Is this OK?

Reply:
Yep sounds great